linux/README.md

```diff
@@ -201,14 +201,14 @@ All packages can be installed on Amazon Linux, CentOS, Fedora, Red Hat Enterpris
 
 | OpenJDK                  | Amazon  | CentOS  | Fedora     | RHEL    | OpenSUSE   | SLES   |
 |--------------------------|---------|---------|------------|---------|------------|--------|
-| JDK 8 (Hotspot, OpenJ9)  | 1, 2    | 6, 7, 8 | 30, 31, 32 | 6, 7, 8 | 15.1, 15.2 | 12, 15 |
-| JDK 9 (Hotspot, OpenJ9)  | 1, 2    | 6, 7, 8 | 30, 31, 32 | 6, 7, 8 | 15.1, 15.2 | 12, 15 |
-| JDK 10 (Hotspot, OpenJ9) | 1, 2    | 6, 7, 8 | 30, 31, 32 | 6, 7, 8 | 15.1, 15.2 | 12, 15 |
-| JDK 11 (Hotspot, OpenJ9) | 1, 2    | 6, 7, 8 | 30, 31, 32 | 6, 7, 8 | 15.1, 15.2 | 12, 15 |
-| JDK 12 (Hotspot, OpenJ9) | 1, 2    | 6, 7, 8 | 30, 31, 32 | 6, 7, 8 | 15.1, 15.2 | 12, 15 |
-| JDK 13 (Hotspot, OpenJ9) | 1, 2    | 6, 7, 8 | 30, 31, 32 | 6, 7, 8 | 15.1, 15.2 | 12, 15 |
-| JDK 14 (Hotspot, OpenJ9) | 1, 2    | 6, 7, 8 | 30, 31, 32 | 6, 7, 8 | 15.1, 15.2 | 12, 15 |
-| JDK 15 (Hotspot, OpenJ9) | 1, 2    | 6, 7, 8 | 30, 31, 32 | 6, 7, 8 | 15.1, 15.2 | 12, 15 |
+| JDK 8 (Hotspot, OpenJ9)  | 1, 2    | 6, 7, 8 | 31, 32, 33 | 6, 7, 8 | 15.1, 15.2 | 12, 15 |
+| JDK 9 (Hotspot, OpenJ9)  | 1, 2    | 6, 7, 8 | 31, 32, 33 | 6, 7, 8 | 15.1, 15.2 | 12, 15 |
```

Comment:
No longer supporting 30?

Reply:
Has reached its EOL a few months ago, too. The Fedora Wiki does not specify an exact date, but it's no longer on the list of supported distributions. See https://fedoraproject.org/wiki/Releases.